### PR TITLE
Fix USD Composer symlink

### PR DESCRIPTION
**Pull request by lilleyse**
https://github.com/CesiumGS/cesium-omniverse/pull/426/ added the material graph extension to our development kit app. Since material graph is only included in certain apps like USD Composer and not kit-sdk we have to locate USD Composer and create a symlink to it. The CMake code to create the symlink was broken.

### CODE REVIEW

File: extern/CMakeLists.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ endif()
 # Add a symlink to USD Composer (create) so that we can use its extensions (e.g. omni.kit.window.material_graph) in our internal applications
 # Don't check for errors because we can still proceed with the build
```
Comment (weegeekps) [inline extern/CMakeLists.txt:52]: We should check for errors, because while the build may still pass if this fails, we can't run the test app.

Comment (weegeekps) [reply extern/CMakeLists.txt:52]: Can we at least log a clear warning to the build output then? This feels like a major footgun.

Comment (lilleyse) [reply extern/CMakeLists.txt:52]: Yeah I can do that

Comment (lilleyse) [reply extern/CMakeLists.txt]: Done

image

**Review by weegeekps — APPROVED** [review]
Works good. A side effect of this is that we do have to have the OV launcher running in the background again, but that's not a terrible thing.

**Pull request merged**